ansible/roles/common/tasks/main.yml

```diff
@@ -124,5 +124,11 @@
   tags:
     - antivirus
     - after-reboot
-- include: cron.yml 
+
+- include: cron.yml
   tags: cron
+
+- include: mount.yml
```

Comment:
Sorry for post merge comment but `include` is deprecated. You should use `import_tasks`